Dockerfile

```diff
@@ -1,12 +1,12 @@
-FROM alpine:3.3
+FROM alpine:edge
 MAINTAINER ilari.makela@wunderkraut.com
 
 # Update the package repository and install applications
 RUN echo http://dl-cdn.alpinelinux.org/alpine/edge/testing >> /etc/apk/repositories && \
```

Comment:
@james-nesbitt yes because you can only install PHP 7 from the testing repo which is under edge.